### DISCUSSION TITLE
Inpainting with image prompts

### DISCUSSION THREAD

**Discussion by carlmoss22** [lllyasviel/Fooocus#963] [Ideas]
I think it would be a great combination if you could use  an image for impainting (like image prompts) instead of text prompts.

**Comment by kubilaykilinc** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/18080130/27e4b0c1-d5eb-4a22-90f4-30c80fc1a090)

**Comment by Phrixus2023** [comment]
> I think it would be a great combination if you could use an image for impainting (like image prompts) instead of text prompts.

i agree with you，like this https://github.com/lkwq007/stablediffusion-infinity
